relational_operators/DropTableOperator.hpp

```diff
@@ -48,9 +48,9 @@ class DropTableOperator : public RelationalOperator {
    * @param only_drop_blocks If true, only drop the blocks belonging to
    *        relation, but leave relation in the database.
    **/
-  explicit DropTableOperator(const relation_id rel_id,
+  explicit DropTableOperator(const CatalogRelation &relation,
```

Comment:
No need to pass `CatalogRelation` to this operator. The original argument `relation_id` is good enough to generate the `work order`.